src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizeNamesFormatter.java

```diff
@@ -36,6 +44,75 @@ public String getKey() {
 
     @Override
     public String format(String value) {
+        Objects.requireNonNull(value);
+        // Handle case names in order lastname, firstname and separated by ","
+        // E.g., Ali Babar, M., Dingsøyr, T., Lago, P., van der Vliet, H.
+        if (!value.contains(" and ") && !value.contains("{") && !value.contains(";")) {
+            String[] valueParts = value.split(",");
+            // Delete spaces for correct case identification
+            for (int i=0; i < valueParts.length; i++) {
+                valueParts[i] = valueParts[i].trim();
+            }
```

Comment:
If you only need each value in a collection, please use a foreach loop as this is more readable in my opinion.
e.g.:
```
for (String name : allNames) {
    name = name.trim();
}
```